### PR TITLE
docs: add more content to getting started vscode

### PR DESCRIPTION
**Pull request by debs-obrien**

### CODE REVIEW

**Comment by debs-obrien** [comment]
closes: https://github.com/microsoft/playwright/pull/18298

File: docs/src/getting-started-vscode-js.md [hunk 1]
```diff
@@ -11,47 +11,50 @@ Get started by installing Playwright and generating a test to see it in action.
...
 
-<img width="1093" alt="choose browsers for Playwright in VS Code" src="https://user-images.githubusercontent.com/13063165/188664742-371f2321-67a1-4799-99ba-253a125de838.png" />
+<img width="1115" alt="Choose Browsers" src="https://user-images.githubusercontent.com/13063165/197704489-72744c50-81ea-4716-a5f1-52ca801edf1f.png" />
 
 ## Running Tests
 
 You can run a single test by clicking the green triangle next to your test block to run your test. Playwright will run through each line of the test and when it finishes you will see a green tick next to your test block as well as the time it took to run the test.
 
-<img width="1272" alt="Running Tests in VS Code" src="https://user-images.githubusercontent.com/13063165/188641041-e7f49b0e-758c-4154-b719-b873ba58dca4.png" />
+<img width="1114" alt="Run a single test" src="https://user-images.githubusercontent.com/13063165/197712138-f4593c0d-ec7e-4a61-b2cd-59fc2af39c6a.png" />
```
Comment (mxschmitt) [inline docs/src/getting-started-vscode-js.md:32]: custom file icons is not default vscode :p

File: docs/src/getting-started-vscode-js.md [hunk 2]
```diff
@@ -61,51 +64,56 @@ With the VS Code extension you can debug your tests right in VS Code see error m
...
 
 ### Live Debugging
 
 You can modify your test right in VS Code while debugging and Playwright will highlight the selector in the browser. This is a great way of seeing if the selector exits or if there is more than one result. You can step through the tests, pause the test and rerun the tests from the menu in VS Code.
 
 
-<img width="1394" alt="Live debugging in VS Code" src="https://user-images.githubusercontent.com/13063165/188644314-89967ab8-2415-4e55-bbca-b3840d347ca4.png" />
+<img width="1350" alt="Live Debugging in VS Code" src="https://user-images.githubusercontent.com/13063165/197967885-512df81f-12e3-45e5-b90f-42ed0f064eac.png" />
+
+### Debug on different Browsers
```
Comment (mxschmitt) [inline docs/src/getting-started-vscode-js.md:83]:
```suggestion
### Debug in different Browsers
```
I think it fits better

**Review by mxschmitt — APPROVED** [review]

File: docs/src/getting-started-vscode-js.md (hunk 1 above)
Comment (debs-obrien) [reply docs/src/getting-started-vscode-js.md:32]: no but its nicer and clearer for people to see

**Pull request merged**